templates/snmpd.service-Debian.epp

```diff
@@ -0,0 +1,18 @@
+###
```

Comment:
Why is it needed to manage the content of this file ? The default one provided with debian 9 is wrong ?

Reply:
We need to manage this file in order to be able to modify `ExecStart` line. This line allow to modify snmpd verbosity. 
This file replace /etc/default/snmpd which was managed by the module but starting with debian 9 doesn't have any impact on snmpd daemon.